nimbleclient/v1/restclient.py

```diff
@@ -268,6 +281,14 @@ def delete(self, endpoint):
                         self._refresh_connection()
                     else:
                         raise NimOSAPIError(response.json())
+                elif response.status_code == 202:
+                    # Handle async job response
+                    job_response = response.json()
+                    if 'messages' in job_response and job_response['messages'][0]['code'] == 'SM_async_job_id':
+                        logging.debug("Got async job response: %s", json.dumps(job_response['messages'], indent=4))
```

Comment:
hope we are not dumping any sensitive data here.. Please take care of this.. if yes then we may want to remove the logging